frontend/src/components/loginForm.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useContext } from "react";
+
+import userContext from '../contexts/context';
+
+const apiUtil = require('../util/apiUtil')
+
+function Login() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const context = useContext(userContext);
+
+  function handleSubmit(e) {
+    
+    e.preventDefault();
+    localStorage.removeItem("jwtToken");
```

Comment:
Separate the logic and the render stuff - move the `localStorage.removeItem("jwtToken");` to `utils/token` or `utils/login`